Fixing issue with `$$` in file content

divdavem opened the pull request:
If a file to be packaged with gulp-noder contains `$$` (or other special replacement patterns), it is mistakenly replaced with `$` (or some other value), because of the special behavior of the string `replace` method, as documented [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_string_as_a_parameter)

PK1A commented:
LGTM, Maybe just amend the commit message to follow https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md ?

divdavem committed:
fix: `$$` in file content is replaced with `$`

If a file to be packaged with gulp-noder contains `$$` (or other special
replacement patterns), it is wrongly replaced with `$` (or some other
value), because of the special behavior of the string `replace` method,
as documented [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_string_as_a_parameter)

divdavem commented:
@PK1A Thank you for your review!
